astropix.py

```diff
@@ -183,12 +212,27 @@ def close_connection(self):
 # Here we intitalize the 8 DAC voltageboard in slot 4. dacvals are carried over from past 
 # scripts. Default from beam_test.py:
 # Use this: (8, [0, 0, 1.1, 1, 0, 0, 1, 1.035])
-    def init_voltages(self, slot: int, vcal:float, vsupply: float, vthreshold:float, dacvals: tuple[int, list[float]] = (8, [0, 0, 1.1, 1, 0, 0, 1, 1.4])):
+    def init_voltages(self, slot: int = 4, vcal:float = .908, vsupply: float = 2.7, vthreshold:float = None, dacvals: tuple[int, list[float]] = None):
+        """
+        Configures the voltage board
+        No required parameters. No return.
+
+        slot:int = 4 - Position of voltage board
+        vcal:float = 0.908 - Calibration of the voltage rails
+        vsupply = 2.7 - Supply Voltage
+        vthreshold:float = None - ToT threshold value. Takes precedence over dacvals if set
+        dacvals:tuple[int, list[float] - vboard dac settings. Must be fully specified if set. 
+        """
+        default_vdac = (8, [0, 0, 1.1, 1, 0, 0, 1, 1.075])
+        
         # used to ensure this has been called in the right order:
         self._voltages_exist = True
-
-        if vthreshold is not None:
-            if vthreshold > 1.7: raise Exception("Threshold voltage out of range!")
+        # Set dacvals
+        if dacvals is None:
+            dacvals = default_vdac
+        # dacvals takes precidence over vthreshold
+        elif vthreshold is not None:
+            if vthreshold > 1.5 or vthreshold < 0: logging.warning("Threshold voltage out of range of sensor!")
```

Comment:
Why change the upper limit from 1.7 to 1.5?

Reply:
The chip fully depletes at 1.5 so I am letting the user know in case they made a typo and are wondering why there's no signal

Reply:
I'm not really sure what you mean by "chip fully depletes at 1.5". Sometimes it's good to set Vth to a very high value just for testing, but throwing a warning in this case seems to be a good idea.

Reply:
I think she's referring to the saturation limit of the PMOS source follower